StRoot/StGmtClusterMaker/StGmtClusterMaker.cxx

```diff
@@ -0,0 +1,302 @@
+//
+// First Cluster Maker
+// \class StGmtClusterMaker
+// \authors K.S. Engle and Richard Witt (witt@usna.edu)
+// based on StFgtClusterMaker
+
+#include "StGmtClusterMaker.h"
+#include "StRoot/StEvent/StEvent.h"
+#include "StRoot/StEvent/StGmtCollection.h"
+//#include "StRoot/StEvent/StTpcHitCollection.h"
+#include "StEvent/StGmtHit.h"
+#include "StRoot/StGmtUtil/geometry/StGmtGeom.h"
+#include "TSystem.h"
+#include "TSpectrum.h"
+#include "StMessMgr.h" 
+ClassImp(StGmtClusterMaker);
+
+Int_t StGmtClusterMaker::gmtStat = 0;
```

Comment:
STAR coding standards prefer using standard c++ types (int, float, char, etc...) over ROOT types (Int_t, Float_t, etc...) when you are implementing a non persistent class... i.e. anything you don't intend save in MuDst, PicoDst, StEvent... or any ttree.

Reply:
Corrected